featuretools/entityset/entityset.py

```diff
@@ -1190,43 +1190,37 @@ def add_interesting_values(self, max_values=5, verbose=False, dataframe_name=Non
             dataframes = [self[dataframe_name]]
         else:
             dataframes = self.dataframes
+
+        def add_value(df, col, val, verbose):
+            if verbose:
+                msg = "Column {}: Marking {} as an "
+                msg += "interesting value"
```

Comment:
should fit on one line now

Reply:
Indeed. Updated both instances of this. Also updated this in the interesting values refactor PR which touches these lines.

Reply:
Updated